__tests__/app.test.js

```diff
@@ -181,6 +192,25 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 
+  it("200: responds with an article object with a comment_count property", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toMatchObject({
+          article_id: 1,
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+          comment_count: 11,
+        });
+      });
+  });
```

Comment:
Nice test. A lot of this may be duplicate to a previous test for get /api/articles/:article_id, the only behaviour we need to test is to ensure that the article we get back now has a new comment_count property